canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -18,131 +18,194 @@
-		display: flex;
-		justify-content: center;
-		user-select: none;
-
-		&.overflow {
-			height: auto;
-			width: 200px;
-			display: block;
+		& button:disabled:hover {
+			background-color: $ui-01;
 		}
 	}
 
-	.list-item-disabled svg {
-		fill: $disabled-02;
+	&.tertiary {
+		& button {
+			color: $brand-03;
+			background-color: $ui-01;
+		}
+
+		& button:hover {
+			background-color: $hover-field;
+		}
+
+		& button:disabled,
+		& button:disabled:hover {
+			color: $disabled-02;
+			background-color: $ui-01;
+		}
 	}
 }
 
-.toolbar-divider {
-	border-right: $toolbar-divider-width solid $ui-03;
+.toolbar-spacer {
+	width: 0;
+	display: inline-block;
 }
 
-.overflow-toolbar-divider {
-	border-bottom: $toolbar-divider-width solid $ui-03;
+.toolbar-overflow-item {
+	background-color: $ui-01;
+	display: inline-block;
+
+	& button {
+		padding: 0;
+		border-top: 0;
+		border-bottom: 0;
+		min-height: 30px;
+		background-color: $ui-01;
+		border-right: $toolbar-divider-width solid $ui-03;
+		// Reduce width by 1 to hide the right border for the rightmost overflow icon in the leftbar.
+		width: calc(#{$toolbar-button-width} - 1px);
+	}
+
+	& button:hover {
+		background-color: $hover-field;
+	}
+
+	& button:disabled:hover {
+		background-color: $ui-01;
+	}
 }
 
-.toolbar-item {
-	// set height/width for tooltip
-	height: 34px;
-	width: 34px;
+.toolbar-overflow-menu-item button {
+	width: 100%;
+	padding-top: 2px;
+	padding-bottom: 2px;
+	border: 0;
+	min-height: 20px;
+}
 
+.toolbar-item-content {
+	width: inherit;
 	display: flex;
 	justify-content: center;
+	padding: 12px 12px 8px;
+
 	&.overflow {
-		width: inherit;
-		padding: 8px;
+		justify-content: left;
+		padding: 8px 0;
+	}
+
+	&.default {
+		color: $icon-01; // Used by label and pick up as currentColor in icons.
+	}
 
-		svg {
-			margin-right: 8px;
+	&.disabled.default {
+		color: $disabled-02;
+	}
+
+	.toolbar-icon {
+		& svg {
+			height: $toolbar-icon-size;
+			width: $toolbar-icon-size;
+			/* Prevent elements from being dragged onto canvas */
+			-webkit-user-drag: none;
 		}
 	}
 }
 
-.toolbar-icons, .toolbar-item > svg {
-	height: $toolbar-icon-size;
-	width: $toolbar-icon-size;
-	margin: auto;
-	/* Prevent elements from being dragged onto canvas */
-	-webkit-user-drag: none;
+.toolbar-text-content {
+	position: absolute;
+	top: 15px;
+	font-size: 9px;
+	line-height: 9px;
+	color: $icon-01;
 }
 
-.overflow-toolbar-icon-label {
-	color: $text-01;
-	width: 100%;
-	word-break: break-word;
-	hyphens: auto;
+.toolbar-icon-label {
+	height: 16px;
+	&.before {
+		padding-right: 8px;
+	}
 
-	&.disabled {
-		color: $disabled-02;
+	&.after {
+		padding-left: 8px;
+	}
+
+	&.overflow {
+		padding-left: 8px;
+		word-break: break-word;
+		hyphens: auto;
 	}
 }
 
+.toolbar-divider {
+	border-right: $toolbar-divider-width solid $ui-03;
+	background-color: $ui-01;
+	display: inline-block;
+	height: $toolbar-button-height;
+	width: 0;
+}
+
+.toolbar-divider-overflow {
```

Comment:
<img width="241" alt="image" src="https://user-images.githubusercontent.com/1628100/88722496-2deb4f80-d0dc-11ea-8310-6d2c3a7ce576.png">
The height will need to be `0` in this case

Reply:
Done